requirements.txt

```diff
@@ -0,0 +1,9 @@
+pillow
+numpy
+tensorflow
+tensorflow-gpu
+python-utils
+scipy
+scikit-learn
+scikit-image
+pyyaml
```

Comment:
where does the `pyyaml` dep come in from?

Reply:
doesn't `tensorflow-gpu` imply `tensorflow` ?

Reply:
> where does the `pyyaml` dep come in from?

File `util.py`

> doesn't `tensorflow-gpu` imply `tensorflow` ?

Well yes, but only if you use a GPU. For people like me, who do not have a powerful GPU (yet) and hence use the CPU, `tensorflow` is the right one...

Reply:
oh yes, util.py, sorry was just grepping in the wrong directory :/